pkg/apis/config/feature_flags.go

```diff
@@ -188,7 +185,7 @@ var (
 // FeatureFlags holds the features configurations
 // +k8s:deepcopy-gen=true
 type FeatureFlags struct {
-	DisableAffinityAssistant         bool `json:"disableAffinityAssistant,omitempty"`
+	// DisableAffinityAssistant         bool `json:"disableAffinityAssistant,omitempty"`
```

Comment:
Either we keep it as no-op (in order to not fail existing configuration), or we remove it, but we shouldn't keep the comment.

Reply:
It seems that the strange annotations generated by `hack/update-codegen.sh` are due to the comments below. I removed the comments and regenerated it, and it worked fine.

Please have a look when you have time. Thank you!